R/read.R

```diff
@@ -1,4 +1,4 @@
-st_read_agnostic <- function(dir, start_year, sectors, risk_type) {
+st_read_agnostic <- function(dir, start_year, sectors, risk_type, start_year_two) {
```

Comment:
should be removed with file renaming Scenarios_AnalysisInput_2022.csv

Reply:
wrong...